tdvt/tdvt/exprtests/regexcalcs/setup.string.regex.icu_fallback.txt

```diff
@@ -23,8 +23,8 @@ REGEXP_EXTRACT_NTH([str1],'(\w+)\s{1}+(\w+)\Z',1)
 REGEXP_MATCH('0123456789', '(?=\d{10})\d{5}')
 REGEXP_REPLACE('0123456789', '(?=\d{10})\d{5}','NewString') 
 REGEXP_MATCH('iamastring', '(?!test)iam\w+')
-REGEXP_MATCH('B���T�F�L100','(?<=B���T�F�L)\d{3}')
-REGEXP_REPLACE('B���T�F�L100B���T�F�L','(?<=B���T�F�L)\d{3}','Replaced')
+REGEXP_MATCH('BÊÂÙTÌFÜL100','(?<=BÊÂÙTÌFÜL)\d{3}')
```

Comment:
Comparing the different expected files. Are you sure we shouldn't just update them both after this change?  It seems like this is going to change the inputs in all cases now.

Reply:
Agreed, I'll update after getting the results of a composite StandardDBs run.

Reply:
I removed all other expecteds (e436c01f0c3caeb5a44976822f62a025e85855f1), added `hyper` and `hyper_native` to the standard suite for Mac & Linux, and run StandardDBs - everything passes.